src/popup/vault/view-custom-fields.component.html

```diff
@@ -19,9 +20,14 @@
                     <span class="sr-only">{{field.value}}</span>
                 </div>
             </div>
-            <div class="action-buttons">
+            <div class="action-buttons" [ngClass]="{'count': field.showCount}">
+                <a class="row-btn" href="#" appStopClick appA11yTitle="{{'toggleCount' | i18n}}"
+                    *ngIf="field.showActionButtons && field.type === fieldType.Hidden && cipher.viewPassword"
+                    (click)="toggleFieldCount(field)">
+                    <i class="fa fa-lg fa-bars" aria-hidden="true"></i>
```

Comment:
Should be the icon with the numbers on it:
```suggestion
                    <i class="fa fa-lg fa-list-ol" aria-hidden="true"></i>
```